warehouse/query-core/src/test/java/datawave/query/AnyFieldQueryTest.java

```diff
@@ -1401,4 +1462,413 @@ protected void testInit() {
         this.auths = CitiesDataType.getTestAuths();
         this.documentKey = CityField.EVENT_ID.name();
     }
+
+    private static class DelayedClient implements AccumuloClient {
```

Comment:
Could be useful as a documented, top-level class?

Reply:
definitely could be used for that. I'd like that to be a separate effort.